client/src/components/reviewsSubComponents/ReviewTile.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+import RatingStars from 'react-star-ratings';
+
+const TileContainer = styled.div`
+  margin: 0 auto;
+  padding: 10px 30px;
+  border-bottom: 1px solid black;
+`;
+
+const monthNames = ['January', 'February', 'March', 'April', 'May', 'June',
+  'July', 'August', 'September', 'October', 'November', 'December'];
+
+const StarDateWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+
+
+
+class ReviewTile extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state ={
+
+    }
+  }
+
+  render() {
+    // Formatting date
+    let date = new Date(this.props.review.date);
+    let month = monthNames[date.getUTCMonth()];
+    let day = date.getUTCDate();
+    let year = date.getUTCFullYear();
+
+
+    return (
+      <TileContainer>
+        <StarDateWrapper>
+          <label><RatingStars
```

Comment:
I would confirm with staff that this outside package is ok here. They may deem it to be cheating